kolibri/core/bookmarks/api.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.status import HTTP_200_OK
+from rest_framework.status import HTTP_201_CREATED
+from rest_framework.status import HTTP_204_NO_CONTENT
+from rest_framework.status import HTTP_400_BAD_REQUEST
+from rest_framework.status import HTTP_403_FORBIDDEN
+from rest_framework.status import HTTP_405_METHOD_NOT_ALLOWED
+
+from .models import Bookmark
+from .serializers import BookmarksSerializer
+from kolibri.core.api import ValuesViewset
+
+
+class BookmarksViewSet(ValuesViewset):
+    values = ("channel_id", "contentnode_id", "id", "content_id")
+    serializer_class = BookmarksSerializer
+    permission_classes = [IsAuthenticated]
+
+    def list(self, request):
```

Comment:
You can avoid a lot of this reimplementation here by using the permissions system and permissioned filtering.

See here for the log model permissions: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/logger/models.py#L61

Obviously would not need the anonymous permission here, and could possibly just use `IsOwn` until there's a use case for admins looking at other people's bookmarks.

You would set this on the model class, and then you could use: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/logger/api.py#L129
```
    permission_classes = (KolibriAuthPermissions,)
    filter_backends = (KolibriAuthPermissionsFilter, DjangoFilterBackend)
```

In order to ensure the default filtering by user is happening.